### DISCUSSION TITLE
Data Types when adding or editing a column

### DISCUSSION THREAD

**Discussion by SlobberingWolfhound** [pawelsalawa/sqlitestudio#4712] [General]
Yesterday I upgraded from SQLiteStudio 3.4.1 to 3.4.3.
This morning I added a new column and noticed that the Data Types in the dropdown were missing several types. 
I am only seeing BLOB, INTEGER, NUMERIC, REAL & TEXT.  
I no longer see BIGINT, BOOLEAN, CHAR, DATE, DATETIME, DECIMAL, DOUBLE, INT, NONE, STRING, TIME or VARCHAR.
Is there a setting I need to change?

**Comment by pawelsalawa** [comment]
The list of types is standarized now, but you still can still enter any type you want (even "XYZ" type if you wish) by just typing it with keyboard in the datatype input.

**Reply by SlobberingWolfhound** [reply]
That is what I was suspecting. Thanks.
